### PR TITLE
Finish English glossary

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR finishes the English glossary and fixes a small issue in the German glossary.

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -2063,12 +2073,40 @@
...
                 {
                     "term": "Signature",
                     "description": "A digital signature is a signature that is created by a crytographic function from a digital document. Digital signatures provide the following properties: <ul><li>Non-repudiation: The issuer cannot retrospectively deny signing the document.</li><li>Integrity: Altering the document invalidates the signature.</li><li>Authenticity: It is clear who signed the document.</li></ul> <br/><br/> As a consequence, the signature must fulfill the following requirements: <br/><ul><li>The signature must be computationally verifiable </li><li>Verifying the signature must be easy - forging a valid signature must be hard</li><li>The signature is only valid for the signed document and no other documents </li><li>The issuer must not be able to deny signing the document.</li></ul>"
                 }
-            ]
+            ],
+            "T": [
+                {
+                    "term": "Test certificate",
+                    "description": "A test est certificate is a => digital certificate issued to a person who has completed a COVID-19 test with a negative result."
```
Comment (MikeMcC399) [inline src/data/faq.json:2097]: "est" should be removed.

File: src/data/faq.json [hunk 2]
```diff
@@ -2063,12 +2073,40 @@
...
+                },
+                {
+                    "term": "Vaccination certificate",
+                    "description": "The vaccination certificate is a => digital certificate issued to a person who has perceived a vaccination with a vaccine approved in the EU. Each vaccination certificate documents exactly one vaccination event (one dose). <br/><br/>The vaccination certificate represents a digital form of a => proof of vaccination."
```
Comment (MikeMcC399) [inline src/data/faq.json:2107]: "perceived" => "received"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Please review individual comments, particularly the invention of the term "professional validity" instead of "effective vaccination validity".

**Comment by Ein-Tim** [comment]
Thank you @MikeMcC399!

I have taken your suggestion into account with https://github.com/corona-warn-app/cwa-website/pull/1543/commits/ca839c91bc7239d06157b7605f629f25985c48e0.

File: src/data/faq.json (hunk 2 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Done in https://github.com/corona-warn-app/cwa-website/pull/1543/commits/ca839c91bc7239d06157b7605f629f25985c48e0

File: src/data/faq.json [hunk 3]
```diff
@@ -2063,12 +2073,40 @@
...
+                }
+            ],
+            "V": [
+                {
+                    "term": "Validity of an EU digital COVID certificate",
+                    "description": "We basically distinguish<ul><li>the validity of the digital certificate (the technical validity) and</li><li>the validity of the vaccination protection (the professional validity).</li></ul>The digital certificate has an issue date and a technical expiration date (so does your passport and PA). The validity of the certificate ends 1 year after issuance. <br/><br/> <b>Vaccination Certificate</b> <br/><br/>In the certificate data is transported, in this case information about vaccination. This vaccination took place on a date X and is valid from then (according to the current state of scientific knowledge) e.g. in Germany 1 year. After that the vaccination protection has expired. We call the duration of the vaccination protection 'professional' validity.So the professional validity of the vaccination counts from the last vaccination date (date X).  <br/><br/> A certificate is valid until either the technical validity or the professional validity expires."
```
[the remark this answers is not in the document]
Comment (Ein-Tim) [reply src/data/faq.json]: Done in https://github.com/corona-warn-app/cwa-website/pull/1543/commits/ca839c91bc7239d06157b7605f629f25985c48e0.

File: src/data/faq.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Done in https://github.com/corona-warn-app/cwa-website/pull/1543/commits/ca839c91bc7239d06157b7605f629f25985c48e0.

File: src/data/faq.json [hunk 4]
```diff
@@ -2043,6 +2043,16 @@
...
                 }
             ],
+            "C": [
+                {
+                    "term": "Check-App / Checking a certificate / Verifying a certificate",
+                    "description": "The mere existence of a (digital) document has no verification/evidence power. Rather, <b>each document/certificate must</b> also be checked for validity and validation. <br/><br/> This check is done by a special app (in Germany the CovPassCheck app), the check app. <br/><br/> Individual countries may have different implementations of the check app. But all check apps use the same - consolidated on a European gateway - keys of the participating countries as well as the stored => Business Rules. Therefore, all check apps will be able to validate all EU certificates in the future, as long as they follow the specification of the European eHealth Network."
```
Comment (heinezen) [inline src/data/faq.json:2049]: maybe better : check app --> verifier app

Comment (heinezen) [reply src/data/faq.json:2049]: Don't start a sentece with "but". Use "However, ..."

File: src/data/faq.json [hunk 5]
```diff
@@ -2063,12 +2073,40 @@
                     "description": "See DCC"
                 }
             ],
+            "P": [
+            {
+                "term": "Proof of vaccination",
+                "description": "The proof of vaccination documents a vaccination procedure with a vaccine recommended by the Paul-Ehrlich-Institut. A vaccination proof must contain at least the following attributes: <ul><li>the personal data of the vaccinated person (at least surname, first name and date of birth)</li><li>date of vaccination, number of vaccinations,</li><li>designation of the vaccine,</li><li>name of the disease against which the vaccination was carried out, and</li><li>characteristics that indicate the person or institution responsible for carrying out the vaccination or issuing the proof of vaccination, for example, an official symbol or the name of the issuer. </li></ul> <br/><br/> For entry, according to § 2 number 10 Coronavirus-Einreiseverordnung, the vaccination certificate must still meet additional requirements (regarding languages and vaccination schema). <br/><br/> The proof of vaccination can be documented in different ways, one form is the digital => vaccination certificate."
```
Comment (heinezen) [inline src/data/faq.json:2079]: Paul-Ehrlich-Institut --> Paul Ehrlich Institute

File: src/data/faq.json [hunk 6]
```diff
@@ -2063,12 +2073,40 @@
...
+            }
+            ],
+            "R": [
+                {
+                    "term": "Recovery Certificate",
+                    "description": "The certificate of recovery is a => digital certificate issued to a person who has demonstrably recovered from COVID-19 and proved immunity by a PCR test."
```
Comment (heinezen) [inline src/data/faq.json:2085]: issued to --> issued for

File: src/data/faq.json [hunk 7]
```diff
@@ -2063,12 +2073,40 @@
...
                 {
                     "term": "Signature",
                     "description": "A digital signature is a signature that is created by a crytographic function from a digital document. Digital signatures provide the following properties: <ul><li>Non-repudiation: The issuer cannot retrospectively deny signing the document.</li><li>Integrity: Altering the document invalidates the signature.</li><li>Authenticity: It is clear who signed the document.</li></ul> <br/><br/> As a consequence, the signature must fulfill the following requirements: <br/><ul><li>The signature must be computationally verifiable </li><li>Verifying the signature must be easy - forging a valid signature must be hard</li><li>The signature is only valid for the signed document and no other documents </li><li>The issuer must not be able to deny signing the document.</li></ul>"
                 }
-            ]
+            ],
+            "T": [
+                {
+                    "term": "Test certificate",
+                    "description": "A test certificate is a => digital certificate issued to a person who has completed a COVID-19 test with a negative result."
```
Comment (heinezen) [inline src/data/faq.json:2097]: issued to --> issued for

File: src/data/faq.json [hunk 8]
```diff
@@ -2063,12 +2073,40 @@
...
+                },
+                {
+                    "term": "Vaccination certificate",
+                    "description": "The vaccination certificate is a => digital certificate issued to a person who has received a vaccination with a vaccine approved in the EU. Each vaccination certificate documents exactly one vaccination event (one dose). <br/><br/>The vaccination certificate represents a digital form of a => proof of vaccination."
```
Comment (heinezen) [inline src/data/faq.json:2107]: issued to --> issued for

**Review by heinezen — CHANGES_REQUESTED** [review]

**Comment by heinezen** [comment]
I have a few remarks. Otherwise, it is good to go.

**Comment by Ein-Tim** [comment]
Thank you @heinezen. I changed what you requested in https://github.com/corona-warn-app/cwa-website/pull/1543/commits/529b7e25f68173e5ffeabe6c3968492bb6d8f967.

File: src/data/faq.json (hunk 8 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Done via https://github.com/corona-warn-app/cwa-website/commit/529b7e25f68173e5ffeabe6c3968492bb6d8f967.

File: src/data/faq.json (hunk 7 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Done via https://github.com/corona-warn-app/cwa-website/commit/529b7e25f68173e5ffeabe6c3968492bb6d8f967.

File: src/data/faq.json (hunk 6 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Done via https://github.com/corona-warn-app/cwa-website/commit/529b7e25f68173e5ffeabe6c3968492bb6d8f967.

File: src/data/faq.json (hunk 5 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Done via https://github.com/corona-warn-app/cwa-website/commit/529b7e25f68173e5ffeabe6c3968492bb6d8f967.

File: src/data/faq.json (hunk 4 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Changed via https://github.com/corona-warn-app/cwa-website/commit/529b7e25f68173e5ffeabe6c3968492bb6d8f967.

**Comment by heinezen** [comment]
@Ein-Tim d5433a1 is a good candidate for amendment to the previous commit ;)

Btw, you can just write the short version (7 characters) of the hash like this 529b7e2 and it will still get properly linked. No need to put the full link. Which version of writing the hash is better in depends on the people you work with. If they checkout the commit locally, the short version is usually more handy.

----
Corona-Warn-App Open Source Team

**Comment by Ein-Tim** [comment]
> https://github.com/corona-warn-app/cwa-website/pull/1543/commits/d5433a14f22dbaf1a6c5cb97af296f007d006f46 is a good candidate for amendment to the previous commit ;)

True, sorry I just saw the build failing and changed it. The next time I will (try to) amendment my commit (:

Thanks for the note with how to link to the commit in the best way (:

**Review by heinezen — APPROVED** [review]

**Pull request merged**